Directory.Packages.props

```diff
@@ -8,6 +8,7 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
     <TestcontainersPackageVersion>4.0.0</TestcontainersPackageVersion>
     <AzureProvisiongVersion>1.0.0</AzureProvisiongVersion>
+    <Npgsql8Version>8.0.6</Npgsql8Version>
```

Comment:
NIT: might be good to add a comment above this property to say that this version should never be updated to 9.0 or greater, since different people will be running tools to update versions and it would be good to make sure we don't update this one passed that. Of course you have the property name saying it is 8 anyway, so if you think that's enough that is good too. For sure don't reset CI just for this, so only consider in case you want to make other changes.

Reply:
Added a comment.